src/View/Helper/ListsHelper.php

```diff
@@ -558,10 +608,21 @@ public function displayListsModule($listsArray)
                         $list['id']
                     )
                 );
+                if(currentUser::isMember()){
```

Comment:
`currentUser` should be capitalized into `CurrentUser`. The code works because PHP is not case sensitive, but let’s try to keep case consistent. :slightly_smiling_face: